Automattic/woocommerce-payments
# [Card Present Payments] Add accounts endpoint

Fixes #1402 

#### Changes proposed in this Pull Request

- Forwards GET requests to `/wp-json/wc/v3/payments/accounts` to WordPress.com `sites/SITEID/wcpay/accounts` after adding the `test_mode` flag if needed

#### Testing instructions

- Upload the following test plugin and ensure you see the response in the notices for the endpoint
```php
<?php
/*
Plugin Name: WCPay Test
*/

function wcpay_test_notice() {
?>
	<div class="notice">
		<span class="wcpaytest">Loading...</span>
	<script>
		jQuery( document ).ready( function() {
			jQuery(".wcpaytest").html( "Requesting..." );
			jQuery.ajax({
				url: "/wp-json/wc/v3/payments/accounts",
				method: 'GET',
				beforeSend: function ( xhr ) {
		 			xhr.setRequestHeader( 'X-WP-Nonce', wpApiSettings.nonce );
				}
			})
			.done( function( data ) {
				var dataAsJSON = JSON.stringify( data );
				jQuery(".wcpaytest").html( 'Success: ' + dataAsJSON );
			})
			.fail( function( xhr, textStatus, msg ) {
				jQuery(".wcpaytest").html( 'Error: ' + textStatus + ' ' + msg );
			});
		});
		</script>	
	</div>
<?php
}

add_action( 'admin_notices', 'wcpay_test_notice' );

```
-------------------
